hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/sigs/order/SigRequirements.java

```diff
@@ -1392,4 +1392,8 @@ private <T> SigningOrderResult<T> topicDelete(
         }
         return factory.forValidOrder(required);
     }
+
+    public SigMetadataLookup getSigMetaLookup() {
+        return sigMetaLookup;
+    }
```

Comment:
Not particularly proud of this, any better ideas? 😆

Reply:
Seems like we have no choice 😁; but we might extend `SigMetadataLookup` with a method like,
```
    SafeLookupResult<AccountSigningMetadata> accountSigningMetaFor(ByteString alias, @Nullable LinkedRefs linkedRefs);
```
so we can avoid constructing `AccountID`'s just for these lookups.

Reply:
Added!

Reply:
Actually, no longer needed with the newest requirements of finalizing only required accounts for the txn 😎